examples/guided.py

```diff
@@ -1,4 +1,4 @@
-import getpass, time, json, os
+import getpass, time, json, sys, signal, os
```

Comment:
What is this code doing here and why is [this](https://github.com/archlinux/archinstall/blob/1be8f1f8cb7e694f4cca19a8f99c43ccc1cd2c01/examples/guided.py#L11-L22) back? How did this not get overwritten by the pull from master? o0

The signal handling and countdown should have been moved into `user_interactions` o0

Reply:
I brought it back because right not it doesn't run without it.

Reply:
I am dropping this commit, but it will need to be fixed elsewhere for 2.2.0 to run.